client/src/app/dashboard/dashboard-overview/dashboard-overview.component.html

```diff
@@ -34,3 +34,19 @@
   </div>
 </div>
 </div>
+
+<!--Notifications Panels-->
```

Comment:
I think it would improve the usability of this feature if the order of notification tables was not fixed.
So If I first expand the crawler notification table and afterwards the webapi table, I would expect the webapi table to be over the crawler's table. 
I'm currently not quite sure how difficult something like this is to implement. If you have an idea how to achieve something like this quickly feel free to add this to the current pull request. Otherwise, I would ask you to open a feature issue so we remember that this might be good to include once we got time.

Reply:
Ok I will have a loo on it and create a ticket if necessary.